apps/docs/components/components/scrollable.md

```diff
@@ -76,64 +99,77 @@ By default `SfScrollable` scroll by one page of items, but can be modified that
 
 ## Props
 
-| Prop name | Type      | Default value | Possible values   |
-|-----------|-----------|---------------|-------------------|
-| `direction` | `SfScrollableDirection`  | `'horizontal'`        | `'horizontal'`, `'vertical'`      |
-| `buttonsPlacement` | `SfScrollableButtonsPlacement`  | `'block'` | `'block'`, `'floating'`, `'none'`      |
-| `activeIndex` | `number`  |  |       |
-| `reduceMotion` | `boolean`  |  |       |
-| `drag` | `{ sensitivity: number; containerWidth: boolean; } | boolean`  |  |       |
-| `prevDisabled` | `boolean`  |  |       |
-| `nextDisabled` | `boolean`  |  |       |
-| `isActiveIndexCentered` | `boolean`  |  |       |
-| `buttonPrevAriaLabel` | `string`  | `'Previous'` |       |
-| `buttonNextAriaLabel` | `string`  | `'Next'` |       |
+| Prop name               | Type                                               | Default value  | Possible values                   |
+| ----------------------- | -------------------------------------------------- | -------------- | --------------------------------- | --- |
+| `direction`             | `SfScrollableDirection`                            | `'horizontal'` | `'horizontal'`, `'vertical'`      |
+| `buttonsPlacement`      | `SfScrollableButtonsPlacement`                     | `'block'`      | `'block'`, `'floating'`, `'none'` |
+| `activeIndex`           | `number`                                           |                |                                   |
+| `reduceMotion`          | `boolean`                                          |                |                                   |
+| `drag`                  | `{ sensitivity: number; containerWidth: boolean; } | boolean`       |                                   |     |
```

Comment:
Im not sure about this line, 
```
`{ sensitivity: number; containerWidth: boolean; } | boolean`
```
is `Type` which is fine but then we have extra column at line 103 which might cause render issue, please can you double check how it looks in docs ? :)

Reply:
@Szymon-dziewonski indeed these auto-linting caused the issue, I removed those changes and disabled auto-fixing on save, shoulld be all good now :)